### ISSUE TITLE
Octave version

### ISSUE THREAD

**Issue by krazedkrish** [epfl-sti/octave-x11-novnc-docker#1]
Which octave  version is it?
Is it 3.x ( >3. 8)
or 4.x( >= 4.0) ?

**Comment by ponsfrilus** [comment]
@krazedkrish it's octave 3.8.1

```
root@8b37547abba6:/scripts# octave -V
GNU Octave, version 3.8.1
```

Octave is installed from the ubuntu repositories (`apt-get install octave`) so it takes the version available here.

Do you know a PPA with octave >= 4.0 ?

**Comment by krazedkrish** [comment]
Thank you.

and here is the ppa link for you.
https://launchpad.net/~octave/+archive/ubuntu/stable

**Comment by ponsfrilus** [comment]
![octave-4 0 0](https://cloud.githubusercontent.com/assets/176002/12919786/fb3d1d76-cf45-11e5-99ee-9187aab115d8.png)

@krazedkrish the latest version is shipped with Octave 4.0.0 from the ppa. The build on [docker hub](https://hub.docker.com/r/epflsti/octave-x11-novnc-docker/) should be ready soon.

**Comment by krazedkrish** [comment]
:+1:
